source/Cache.h

```diff
@@ -0,0 +1,330 @@
+/* Cache.h
+Copyright (c) 2018 by Michael Zahniser
```

Comment:
This is a new file, correct? Update the copyright then. :)

Reply:
It's the first published year, and I would assign MZ the copyright of my part.